docs/reference/tab-widgets/troubleshooting/data/increase-cluster-shard-limit.asciidoc

```diff
@@ -0,0 +1,198 @@
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+PUT _cluster/settings
+{
+  "persistent" : {
+    "cluster.routing.allocation.total_shards_per_node" : null
+  }
+}
+
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+// tag::cloud[]
+In order to get the shards assigned we'll need to increase the number of shards 
+that can be collocated on a node in the cluster.
+We'll achieve this by inspecting the system-wide `cluster.routing.allocation.total_shards_per_node` 
+<<cluster-get-settings, cluster setting>> and increasing the configured value.
+
+Note that this requires the current data nodes to have enough disk space to host 
+all the data.
+If the current deployment configuration doesn't have enough capacity to store
+all the data, including the unassigned shards, please contact 
+https://support.elastic.co[Elastic Support] to discuss available options.
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is 
+{cloud}/ec-access-kibana.html[enable Kibana first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Dev Tools > Console**.
++
+[role="screenshot"]
+image::images/kibana-console.png[{kib} Console,align="center"]
+
+. Inspect the `cluster.routing.allocation.total_shards_per_node` <<cluster-get-settings, cluster setting>> 
+for the index with unassigned shards:
++
+[source,console]
+----
+GET /_cluster/settings?flat_settings
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "persistent": {
+    "cluster.routing.allocation.total_shards_per_node": "300" <1>
+  },
+  "transient": {}
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only as don't want to change a cluster-wide setting]
+
++
+<1> Represents the current configured value for the total number of shards
+that can reside on one node in the system.
+
+. <<cluster-update-settings,Increase>> the value for the total number of shards 
+that can be assigned on one node to a higher value:
++
+[source,console]
+----
+PUT _cluster/settings
+{
+  "persistent" : {
+    "cluster.routing.allocation.total_shards_per_node" : 400 <1>
+  }
+}
+----
+// TEST[continued]
+
++
+<1> The new value for the system-wide `total_shards_per_node` configuration
+is increased from the previous value of `300` to `400`. 
+The `total_shards_per_node` configuration can also be set to `null`, which 
+represents no upper bound with regards to how many primary shards can be 
```

Comment:
```suggestion
represents no upper bound with regards to how many shards can be 
```
This setting allows both primary and replica shards. So you either should include them both, or just drop it for clarity.